index.html

```diff
@@ -47,11 +53,12 @@ <h3 class="black">Hey there. I’m Jemutai.<br><span id="green">I’m a software
                         <a href="#"><img src="images/Vector3.png" alt="social media icons"></a>
                     </li>
                 </ul>
-    </section> 
+            </section> 
+    <!-- </section>  -->
     <!-- work section -->
     <section class="works">
         <img id="action-icon" src="images/Action.png" alt="">
-        <h2 class="text" id="text2">My Recent<br> Works</h2>
+        <h2 class="text" id="text2">My Recent Works</h2>
         <img id="line" src="images/Indicator.png" alt="#">
     <section class="card-workrow">
         <div class="card-work">   
```

Comment:
- [ ] Also in the works section the cards are a little bit to the left, please fix that.

Your Design | Figma Design
---------------- | -------------------
![Screenshot from 2022-08-25 16-42-43](https://user-images.githubusercontent.com/33752735/186697695-bef84819-2d31-4eb7-80c5-25a395b543fc.png) | ![Screenshot from 2022-08-25 16-50-55](https://user-images.githubusercontent.com/33752735/186697801-b25a52b3-dd6e-41ad-86b9-0d7568beecd9.png)
![Screenshot from 2022-08-25 16-43-06](https://user-images.githubusercontent.com/33752735/186697968-271dd10d-6fee-4e10-9d1f-6cc841c7ceb5.png) | ![Screenshot from 2022-08-25 16-52-30](https://user-images.githubusercontent.com/33752735/186698177-8e9b5001-223a-4c72-aa1b-72cd9b28046e.png)

Reply:
- [ ] Please align these cards as requested by the previous reviewer since it is a core requirement for you to follow the design seen on the Figma template 👍